iroha_introspect/iroha_introspect_bin/src/main.rs

```diff
@@ -0,0 +1,34 @@
+//! Binary to print all types to json string
+
+#![allow(clippy::print_stdout)]
+
+use std::collections::BTreeMap;
+
+use iroha_data_model::prelude::*;
+use iroha_data_model::query::QueryBox;
+use iroha_introspect::prelude::*;
+
+macro_rules! set_types {
+    ($($t:ty),* $(,)?) => {{
+        let mut out = BTreeMap::new();
+        $(<$t as Introspect>::introspect(&mut out);)*
+        serde_json::to_string_pretty(&out).unwrap()
+    }};
+}
+
+fn main() {
+    println!(
+        "{}",
+        set_types!(
+            Account,
```

Comment:
I think we do not need to describe many types except high-level types defined in torii:  VersionedEvent,
VersionedTransaction,  VersionedQueryResult, VersionedQueryRequest, VersionedSubscriptionRequest

Reply:
I guess it is better to supply more. This pr is not final, as there is no artifact for ci, and this type list is not full. Thanks